### ISSUE TITLE
Can we drop 1.7 support in v2?

### ISSUE THREAD

**Issue by Dargmuesli** [thelinmichael/spotify-web-api-java#112]
Currently, Java 1.7 is supported, even in v2. This makes a few things trickier and hacks are needed to achieve certain functionality.

E.g. Base64 support. Up to Java 1.8 the `java.xml.bind.DatatypeConverter` can be used. Since Java 1.8 we could use the `java.util.Base64` class. But using both at the same time doesn't work, a dependency for that is a little over the top and thus I added an own simple base64-implementing class.

I don't really like that state.

**TL;DR: Does anyone really need support for 1.7?**

If not, it'll get dropped.

**Comment by seantcanavan** [comment]
We've been running with Java 8 for quite a while now. This would not break anything for us.

**Comment by Dargmuesli** [comment]
We'll keep support for Java 7 for the initial release of v2. Does anyone know if it's possible to deprecate a Java/Maven compiler version? Support for Java 7 will be removed in the future, when v2 runs stable.

**Comment by Dargmuesli** [comment]
Version 2.0.0-RC1 was just released.
